cmp/chart/ChartModel.js

```diff
@@ -16,6 +16,10 @@ export class ChartModel {
 
     @bindable.ref highchartsConfig = {};
     @bindable.ref series = [];
+    /**
+     * The HighCharts chart instance currently being displayed.
```

Comment:
Need the warnings here about:  not mutating this, use for reading state only, use the setHighchartsConfig() if you want to modify the underlying chart, etc.etc.

Reply:
also, we might want to call this `highchart` rather than hchart.